raster3d/r3.in.v5d/v5d.h

```diff
@@ -43,8 +43,8 @@
  * Define our own 1 and 2-byte data types.  We use these names to avoid
  * collisions with types defined by the OS include files.
  */
-typedef unsigned char V5Dubyte;   /* Must be 1 byte, except for cray */
-typedef unsigned short V5Dushort; /* Must be 2 byte, except for cray */
+typedef unsigned char V5Dubyte;   /* Must be 1 byte */
+typedef unsigned short V5Dushort; /* Must be 2 byte */
```

Comment:
I wonder if it wouldn't be better to use C99's `int8_t` and `int16_t` (from stdint.h) here? That should by definition be the same on any platform.

Reply:
I agree. Please, open a separate issue or PR for that. I intend to merge this just with the removal.